.github/workflows/checks.yml

```diff
@@ -23,6 +23,12 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
```

Comment:
It looks like `v4` is the latest major version available for this action, we'll want to use that: https://github.com/actions/setup-node/releases/tag/v4.0.3 - unless there was an issue with it that you found?

Reply:
Good catch, updated.